src/metabase/server.clj

```diff
@@ -33,7 +33,8 @@
             :max-threads   (config/config-int :mb-jetty-maxthreads)
             :min-threads   (config/config-int :mb-jetty-minthreads)
             :max-queued    (config/config-int :mb-jetty-maxqueued)
-            :max-idle-time (config/config-int :mb-jetty-maxidletime)})
+            :max-idle-time (config/config-int :mb-jetty-maxidletime)
+            :request-header-size (config/config-int :mb-jetty-request-header-size)})
```

Comment:
This will require a change to not add to the map if it's not set. Will provide an example.